src/components/screens/WalletKit/DappRequestBottomSheetContent.tsx

```diff
@@ -51,13 +70,94 @@ const DappRequestBottomSheetContent: React.FC<
   onCancelRequest,
   onConfirm,
   isSigning,
+  isMalicious,
+  isSuspicious,
+  transactionScanResult,
+  securityWarningAction,
+  signTransactionDetails,
   isMemoMissing,
   isValidatingMemo,
   onBannerPress,
 }) => {
   const { themeColors } = useColors();
   const { t } = useAppTranslation();
-  const { copyToClipboard } = useClipboard();
+  const transactionBalanceListItems = useTransactionBalanceListItems(
+    transactionScanResult,
+  );
+
+  const accountDetailList = useMemo(
+    () => [
+      {
+        icon: (
+          <Icon.Wallet01 size={16} color={themeColors.foreground.primary} />
+        ),
+        title: t("wallet"),
+        trailingContent: (
+          <View className="flex-row items-center gap-2">
+            <Avatar
+              size="sm"
+              publicAddress={account?.publicKey ?? ""}
+              hasBorder={false}
+              hasBackground={false}
+            />
+            <Text md primary>
+              {account?.accountName}
+            </Text>
+          </View>
+        ),
+        titleColor: themeColors.text.secondary,
+      },
+      {
+        icon: <Icon.Route size={16} color={themeColors.foreground.primary} />,
+        title: t("transactionAmountScreen.details.fee"),
+        trailingContent: (
+          <View className="flex-row items-center gap-2">
+            <Text md primary>
+              {formatAssetAmount(
+                String(signTransactionDetails?.summary.feeXlm ?? "0"),
+                NATIVE_TOKEN_CODE,
+              )}
+            </Text>
+          </View>
+        ),
+        titleColor: themeColors.text.secondary,
+      },
+    ],
+    [account, themeColors, t, signTransactionDetails?.summary.feeXlm],
+  );
+
+  const changeTrustTokenList = useMemo(() => {
+    const changeTrustOp = signTransactionDetails?.operations.find(
+      (op) => op.type === "changeTrust",
+    ) as
+      | { type: "changeTrust"; line?: { code?: string; issuer?: string } }
+      | undefined;
+
+    const assetCode = changeTrustOp?.line?.code;
+    const issuerKey = changeTrustOp?.line?.issuer;
+
+    if (!assetCode || !issuerKey) {
+      return [] as { key?: string; icon: React.ReactNode; title: string }[];
+    }
+
+    const token = { code: assetCode, issuer: { key: issuerKey } } as never;
+
+    return [
+      {
+        key: `${assetCode}:${issuerKey}`,
+        icon: <AssetIcon token={token} size="sm" />,
```

Comment:
@danilosilvackl @leofelix077 I see we are still using the "Asset" nomenclature on this PR, we should review all components/variables/strings on this PR before merging it